src/web/app/pages-session/session-submission-page/saving-complete-modal/saving-complete-modal.component.html

```diff
@@ -35,15 +35,16 @@ <h5 class="modal-title">
     Note that you can change your responses and submit them again any time before the session closes.
   </p>
   <p *ngIf="!hasFailToSaveQuestions">
-    You are encouraged to download the proof of submission, which will also contain your latest responses.
+    <b>You are encouraged to download the proof of submission, which will also contain your latest responses.</b>
   </p>
 </div>
 <div class="modal-footer">
-  <button type="button" class="btn btn-info"
-          (click)="downloadProofOfSubmission()"
-          [disabled]="isAllQuestionSavingFailed">Download proof of submission</button>
   <button type="button" class="btn modal-btn-ok"
```

Comment:
We could add an additional class (like perhaps `btn-info`) to the end here, that should remove the need for writing extra css